ui/Screen/Discovery.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { getFeed } from "../src/discovery.ts";
+
+  import { SidebarLayout } from "../DesignSystem/Component";
+  import { Icon, Input, Title } from "../DesignSystem/Primitive";
+
+  import Project from "./Discovery/Project.svelte";
+
+  const projectFeed = getFeed();
+</script>
+
+<style>
+  .container {
+    margin: 64px 100px;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: minmax(0, 1fr) minmax(0, 1fr);
+    column-gap: 32px;
+    row-gap: 32px;
```

Comment:
I know it's `32px` in the design file, but I think we use `24px` everywhere else. So maybe it's nice to align it. Plus `grid-gap` does the both column and row.
```suggestion
    grid-gap: 24px;
```

Reply:
nice! makes sense. it's tedious to update figma all the time but it'd be cool if the specs were as close to the app as possible.